### PR TITLE
resolved transaction refresh issue and blocks refresh issue fix

### LINKED ISSUE #499: transactions flash every 30s
The transaction service fetches for new transactions every 30s on top of fetching the transactions when tx count has changed. Is it necessary ?

This interval update causes Material to reload the list and displaying an adjustment animation from left to right.

https://github.com/particl/partgui/blob/dev/src/app/wallet/wallet/shared/transaction.service.ts#L54

### PR DESCRIPTION
**Pull request by vikas-cis**
Fixes #499

### CODE REVIEW

**Comment by allienworks** [comment]
Tested and seems to work nicely.
The transactions don't flash anymore, the TX details don't get closed as well.

Still needs code review.

**Comment by kewde** [comment]
The flickering is caused by deleteTransactions().
It deletes all the transactions and then adds the new ones one-by-one.

Might just be resolved by replacing (without deleting) & observables

File: src/app/wallet/wallet/shared/transaction.service.ts [hunk 1]
```diff
@@ -154,7 +168,33 @@ export class TransactionService {
...
+
+  checkForNewTransaction(oldTxs: any, newTxs: any) {
+    const newTransaction = _(newTxs).differenceBy(oldTxs, 'txid').value();
+    newTransaction.map(tx => {
+      this.unShiftTransactions(tx);
+    });
+  }
+
+  unShiftTransactions(json: Object) {
+    if (this.txs.length === 10) {
```
Comment (kewde) [inline src/app/wallet/wallet/shared/transaction.service.ts]: Isn't the length variable depending on the paginator?

Comment (vikas-cis) [reply src/app/wallet/wallet/shared/transaction.service.ts]: yes 'MAX_TXS_PER_PAGE' initially equals to 10

**Comment by pciavald** [comment]
I open the wallet, daemon connects and my balances show up, but not the transactions...
screenshot_20180104_132802

**Comment by allienworks** [comment]
@pciavald running my wallet on testnet, I don't have that issue. I can see the history without problems.

**Comment by pciavald** [comment]
@allienworks it worked for me only the first time, then restarted the wallet and transactions were blank with correct balance

**Comment by vikas-cis** [comment]
@pciavald I will check that today

**Comment by vikas-cis** [comment]
@pciavald can you please test it now

**Pull request merged**